### DISCUSSION TITLE
Self-hosted supabase instance logs page doesn't works

### DISCUSSION THREAD

**Discussion by raman04-byte** [supabase/supabase#27772] [Questions]
I recently self-hosted a Supabase instance by following the documentation: Supabase Self-Hosting Guide.

However, I am unable to access the logs when I navigate to the specified location (as shown in the image).
Do I need to perform any additional configuration steps for this page to work?
Or do I need to host a logflare image along side with it?

Currently, I have executed only the following commands in my terminal:
# Get the code
git clone --depth 1 https://github.com/supabase/supabase

# Go to the docker folder
cd supabase/docker

# Copy the fake env vars
cp .env.example .env

# Pull the latest images
docker compose pull

# Start the services (in detached mode)
docker compose up -d

**Comment by GaryAustin1** [comment]
Not sure what image you refer to.
Did you go thru the docs for self-hosting in particular the analytics server
https://supabase.com/docs/reference/self-hosting-analytics/introduction
Overall docs: https://supabase.com/docs/guides/self-hosting/docker

**Reply by raman04-byte** [reply]
![345395302-f78f56c9-3e36-4183-80ea-5c4cec594faa.png](https://github.com/supabase/supabase/assets/76096922/c98c30d8-75a6-4c62-88f6-c9ea4b693f5f)

Take a look into this image for better clarity. We are getting error in logs

**Reply by raman04-byte** [reply]
And I had followed the steps from the documentation

**Reply by GaryAustin1** [reply]
If you went thru everything in this: https://supabase.com/docs/reference/self-hosting-analytics/introduction
I don't have other advice as I have never tried self-hosting.    You had said you only executed the listed commands in your first post and that link shows lots of other work to get logging to work.

**Reply by raman04-byte** [reply]
> If you went thru everything in this: https://supabase.com/docs/reference/self-hosting-analytics/introduction I don't have other advice as I have never tried self-hosting. You had said you only executed the listed commands in your first post and that link shows lots of other work to get logging to work.


I followed other steps as still I am facing that issue

**Reply by raman04-byte** [reply]
> If you went thru everything in this: https://supabase.com/docs/reference/self-hosting-analytics/introduction I don't have other advice as I have never tried self-hosting. You had said you only executed the listed commands in your first post and that link shows lots of other work to get logging to work.

You can see more information here

https://github.com/supabase/supabase/issues/27990
